### ISSUE TITLE
Private Policy

### ISSUE THREAD

**Issue by BeepDroid** [IanTDuncan/MealTime#62]
For keeping track of the private policy progress and ensuring we are all aware and on board with what is being included in it. Daft updates, discussion, and final version goes here before implementation.

**Comment by BeepDroid** [comment]
Okay, this will have to be finished first I think. I'm following a guide that suggest the private policy is finished BEFORE terms of service so that the former can reference the latter. So far, this is what I've got through the guide. I've followed along with the ad information and what not. 

![image](https://github.com/IanTDuncan/MealTime/assets/71675347/9a972acc-e493-44f1-b384-02f890d93fda)


This is the Key Points before we get into the meat of it. Is there anything wrong with this so far? @aaleksandraristic @Cade5480 @bdshanks-28 @IanTDuncan

**Comment by BeepDroid** [comment]
![image](https://github.com/IanTDuncan/MealTime/assets/71675347/17287f3d-537b-4c99-a0f4-cdaaa42012cc)

I've also done this. I'm using a webstie called Termly which has all the legal layout and details for formatting these things. It asks me questions as I go along and some of them we may have to do together. It's all pretty straight forward.

**Comment by aaleksandraristic** [comment]
This looks great! I wouldn't change anything here. What do you guys think?

**Comment by aaleksandraristic** [comment]
So as I understand the second picture is whenever you click on the link to get more info from the first one, right?

**Comment by Cade5480** [comment]
I think this all looks good too, I don't think we need to change  anything as well!

**Comment by BeepDroid** [comment]
> So as I understand the second picture is whenever you click on the link to get more info from the first one, right?

Yep! That's exactly how it would work, it goes down further in the page to the details that have yet to be fully fleshed out.

**Comment by BeepDroid** [comment]
![image](https://github.com/IanTDuncan/MealTime/assets/71675347/a8d21154-ffe0-413b-9f58-e4c360a85cb2)
![image](https://github.com/IanTDuncan/MealTime/assets/71675347/60a26b0f-1413-45b9-9c99-b798d266f76a)
![image](https://github.com/IanTDuncan/MealTime/assets/71675347/5ed9ceca-34d1-46af-ad40-81b268b2d881)
Some updates from the privacy policy progress! The blank in the last screenshot is simply from not having a solid contact form/information for us collectively as of yet. Any edits I should make?

**Comment by aaleksandraristic** [comment]
This looks good to me, I wouldn't edit anything!

**Comment by BeepDroid** [comment]
https://docs.google.com/document/d/1cw8yiD_NnWHi7fU0eIWfzHyOvrOR9fl434M-Aw5TY-Y/edit?usp=sharing

I added the first draft to this drive! The blanks still present are still just for contact information from our end, since we don't have a central email or anything like that. Take a read and tell me what you think!
@aaleksandraristic @Cade5480 @bdshanks-28 @IanTDuncan

**Comment by BeepDroid** [comment]
I've updated the privacy policy and added a clause for our cookies policy! I have the basic informastion there and am going to add the Firebase specific services we use in a stacked format inside of it once I verify I don't need to put any particular state laws pretaining to cookies.

**Comment by BeepDroid** [comment]
Alright gang, I've updated the privacy policy draft with the framework for the clause following the aknwoldgement of cookies in our app. This section is very bare bones as of now, just blocking out the information I'm going to include in that clause that will make up the bulk of our cookie policy.

**Comment by BeepDroid** [comment]
As the drafts have been reviewed and approved for app inclusion as they are, and the team has already approved closing of this ticket, I'm ending this issue as completed. 

If an update needs to be made in the future, a new ticket will be created for it. 

Final Notes: 

Team has access to link for the entirety of the document. 
Cade has access to document for editing purposes as he is handling app implementation. 
Access for editing can and will be given upon email request from the document.
